Mathlib/NumberTheory/LSeries/RiemannZeta.lean

```diff
@@ -209,7 +209,8 @@ lemma two_mul_riemannZeta_eq_tsum_int_inv_pow_of_even {k : ℕ} (hk : 2 ≤ k) (
     norm_cast
     simp [h0, zeta_eq_tsum_one_div_nat_add_one_cpow (s := k) (by simp [hkk]),
       tsum_pnat_eq_tsum_succ (f := fun n => ((n : ℂ) ^ k)⁻¹)]
-  · simp [Even.neg_pow hk2]
+  · intro n
+    simp [Even.neg_pow hk2]
```

Comment:
Merge error?

Reply:
This was because `tsum_int_eq_zero_add_two_mul_tsum_pnat` now uses `Even` as a hypothesis, not the explicit version it had before.